src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lotter = new Lottery();
+        Ball[] balls = {lotter.getRandomBall(),lotter.getRandomBall(),lotter.getRandomBall()};
```

Comment:
[Use for loop for creating several objects of the same class](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class)